pypdf/_writer.py

```diff
@@ -2733,6 +2739,9 @@ def merge(
                 ].indirect_reference
                 self.add_named_destination_array(dest["/Title"], arr)
 
+        for dest in reader._namedDests.values():
```

Comment:
Why is the name `_namedDests` not yet following PEP8?

Reply:
something missed months/years(?) ago
I will change it tonight